_posts/2021-11-01-r-6.0.0.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: post
+title: Apache Arrow R 6.0.0 Release
+date: "2021-11-01"
+author: 
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+We are happy to announce the recent release of version 6.0.0 of Arrow on CRAN, 
+and in this blog post we highlight the main updates in this version.  A big 
+thanks goes to Dragos Moldovan-Grünfeld, Percy Camilo Triveño Aucahuasi, 
+Dewey Dunnington, Matt Peterson, and Phillip Cloud, who, in this release, made 
+their first contributions the to the R package.
+
+# Grouped aggregation
+
+Aggregations can now be made across groups using dplyr’s `group_by() %>% summarise()` syntax. Arrow 5.0.0 allowed `summarise()` to aggregate across a whole dataset, but 6.0.0 now allows you to aggregate across groups with `group_by()` (a workflow we know people have been waiting for and asking about!). These are usable both with in-memory Arrow tables as well as across partitioned datasets. As usual, Arrow will read and process data in chunks and in parallel when possible to produce results much faster than one could by loading it all into memory then processing, and even better, allows for operations that wouldn’t fit into memory on a single machine.
+
+The focus of this release has been on the initial implementation of this functionality - for the next release, we’ll be looking to profile and optimize to enhance performance.
```

Comment:
```suggestion
The focus of this release has been on the initial implementation of this functionality - for the next release, we’ll be looking to profile and optimize to enhance performance. Connected with that, much of this functionality is still very new and slightly experimental, (for example, it's not even wired up in the pyarrow package yet!). We are excited to have people try this out, if you run into any issues at all, please [let us know](https://issues.apache.org/jira/browse/ARROW) so that we can improve these for our next release.
```